lib/export-notebook.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+
+import * as path from "path";
+import { writeFile } from "fs";
+
+const { dialog } = require("electron").remote;
+const { stringifyNotebook } = require("@nteract/commutable");
```

Comment:
Only a minor note: Since we're using `import` syntax everywhere else it would be cool to stick with it here too.